application/views.py

```
@@ -0,0 +1,49 @@
+from flask import render_template, request
+
+from application import app
+from application.models import Microcontroller, Module
+
+
+def get_parts_list(part_type, category):
```

there's not need to check if the database is working correctly - that's up to the database. you could check if your import script works correctly, though - eg. feed it some dummy data and see if it adds things correctly when it can, and that it raises the appropriate errors when invalid data is provided. in that case you could split the load_xml_to_db.py file into little functions and unit test each one of them. the views would have to have integration tests, but I reckon you can skip them for now. you could also unit test the __repr__  function in the models and make sure that it returns the correct strings for different models